client/components/tinymce/plugins/wpcom-autoresize/plugin.js

```diff
@@ -101,6 +125,13 @@ function wcpomAutoResize( editor ) {
 			if ( tinymce.isWebKit && deltaSize < 0 ) {
 				resize( e );
 			}
+
+			if ( e && e.type === 'keyup' ||
```

Comment:
Thinking out loud since the `isEndOfEditor` loop bothers me:

- Somehow attach the editor to the bottom of the scroll so that when height changes, it grows upward instead of down
- Check scrollbar position relative to bottom and ensure that when height changes that we determine whether `window.scrollY` should be adjusted such this offset is preserved:

```js
const beforeOffset = document.documentElement.clientHeight - window.scrollY;
DOM.setStyle( /* ... */ );
const afterY = document.documentElement.clientHeight - beforeOffset;
if ( afterY !== window.scrollY ) {
	window.scrollTo( 0, afterY );
}
```